### PR TITLE
Remove lock

### PR DESCRIPTION
**Pull request by MonsieurPetion**
Constructor refactoring of room
Lock removed 
Test refactoring 
Code deletationg

### CODE REVIEW

File: app/TDG/RoomTDG.py [hunk 1]
```diff
@@ -22,20 +22,14 @@ def findAll():
...
-    conn = connect_db()
-    if conn:
-	cur = conn.cursor()
-	cur.execute("""INSERT INTO roomTable(lock) VALUES (%s);""", lock)
-	conn.commit()
-	conn.close()
 
-def update(id, availability):
+
+def update(id):
```
Comment (lancelafontaine) [inline app/TDG/RoomTDG.py]: Now that rooms don't have any data other than IDs, updates don't make much sense. You can remove this function altogether

File: app/__init__.py [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
-from flask_cors import CORS, cross_origin
+#from flask_cors import CORS, cross_origin
```
Comment (lancelafontaine) [inline app/__init__.py]: Don't commit these things as commented, leave them uncommented in your local if that's how it works for you

File: app/mapper/RoomMapper.py [hunk 3]
```diff
@@ -30,10 +30,11 @@ def findAll():
             rooms.append(room)
     return rooms
 
-def setRoom(roomId, lock):
+#need to remove do thing
+'''def setRoom(roomId, lock):
     room = find(roomId)
-    room.setLock(lock)
-    update(room.getId(),room.getLock())
+#    room.setLock(lock)
```
Comment (lancelafontaine) [inline app/mapper/RoomMapper.py]: Remove this comment

File: app/mapper/RoomMapper.py [hunk 4]
```diff
@@ -30,10 +30,11 @@ def findAll():
...
+    update(room.getId())'''
```
Comment (lancelafontaine) [inline app/mapper/RoomMapper.py]: Three random apostrophes here, remove

File: app/mapper/RoomMapper.py [hunk 5]
```diff
@@ -42,12 +43,13 @@ def done():
     UnitOfWork.commit()
 
 #adds room object
-def save(room):
-    RoomTDG.insert(room.getLock())
+#def save(room):
```
Comment (lancelafontaine) [inline app/mapper/RoomMapper.py]: Can remove this

File: app/mapper/RoomMapper.py [hunk 6]
```diff
@@ -42,12 +43,13 @@ def done():
...
+  #  RoomTDG.insert(room.getLock())
 
 #updates room Object
-def update(room,availability):
-    RoomTDG.update(room, availability)
+#need to remove do thing
```
Comment (lancelafontaine) [inline app/mapper/RoomMapper.py]: Also remove this

File: app/__init__.py (hunk 2 above)
Comment (MonsieurPetion) [inline app/__init__.py]: i will git ignore this file

File: app/mapper/RoomMapper.py (hunk 3 above)
Comment (MonsieurPetion) [inline app/mapper/RoomMapper.py]: done

File: app/mapper/RoomMapper.py (hunk 4 above)
Comment (MonsieurPetion) [inline app/mapper/RoomMapper.py]: done

File: app/mapper/RoomMapper.py (hunk 5 above)
Comment (MonsieurPetion) [inline app/mapper/RoomMapper.py]: done

File: app/mapper/RoomMapper.py (hunk 6 above)
Comment (MonsieurPetion) [inline app/mapper/RoomMapper.py]: done

File: app/__init__.py (hunk 2 above)
Comment (MonsieurPetion) [inline app/__init__.py]: done

File: app/TDG/RoomTDG.py (hunk 1 above)
Comment (MonsieurPetion) [inline app/TDG/RoomTDG.py]: done

**Comment by choitwao** [comment]
So this PR has the samething as another PR

**Pull request merged**